.env

```diff
@@ -0,0 +1,2 @@
+BACKEND = http://120.222.204.155:30100
```

Comment:
The backend server is `http://klever-model-registry:8080` when in cluster.

Reply:
For klever-web, it will send request to klever-model-registry from browser, maybe it should be klever-model-registry's external address? @hewenguang  WDYT

Reply:
这个文件是前端开发时本地的环境变量，部署容器时是取得后端实际的环境变量，但不是写死的